waltz-ng/client/technology/index.js

```diff
@@ -16,11 +16,15 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 import angular from "angular";
+import {registerService} from '../common/module-utils';
+import * as TechnologyStatisticsService from './services/technology-statistics-service';
 
 
 export default () => {
     const module = angular.module('waltz.technology', []);
 
+    registerService(module, TechnologyStatisticsService);
+
     module
         .directive('waltzTechnologySection', require('./directives/technology-section'));
```

Comment:
perhaps replace the require() calls with imports since the file is being touched